### DISCUSSION TITLE
Renewing my GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by stantios** [community/community#55871] [GitHub Education]
### Hi everyone,

I am here to talk to guys here regarding my issue with most if not all services that I used the github student developer pack to sign up with, are saying that I need to renew my application. When I tried to renew my application / student status, it says there is no need to renew it which is either wrong or the services that I signed up with are wrong. I think that my GitHub student developer pack has run out and that GitHub is telling me it hasn't and that I do not need to resubmit, when I actaully do. Here is the image with the error message I keep getting when trying to renew:

![Screenshot_20230518-203638_Chromium](https://github.com/community/community/assets/98621617/e35f18b0-8c5a-4f25-bbc4-187fc4e689f8)

Thank You!

**Comment by ryanlua** [comment]
Hi @Imzxhir,

You should edit your discussion to have an embed to your image and not the link which is broken. 

I would suggest you [contact GitHub Support](https://support.github.com/contact/education) as it sounds like an error or bug on their end for clarification if you have the student benefits.

Have you removed or unverified any emails for your GitHub account recently?

**Reply by stantios** [reply]
> Hi @Imzxhir,
> 
> You should edit your discussion to have an embed to your image and not the link which is broken.
> 
> I would suggest you [contact GitHub Support](https://support.github.com/contact/education) as it sounds like an error or bug on their end for clarification if you have the student benefits.
> 
> Have you removed or unverified any emails for your GitHub account recently?

Nope, I have not removed any unverified or verified emails recently. When I try to contact GitHub Support about my issue they keep saying to contact the services that I have benefits with which is not an issue on their end, and instead is an issue on GitHub's end. 

EDIT: I have created a new ticket with GitHub Support about my issue and they said that they have recieved my message and to be patient while waiting for a response from them so hopefully that means they will look into the issue and get it fixed.
